pages/mongo/mongo_migration_getting_started.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Getting started with Mongo and data migrations
+summary: Create a brand new project with a migration
+tags: [migration, getting_started]
+keywords: [migration]
+sidebar: mongo_sidebar
+permalink: /mongo_migration_getting_started
+folder: mongo
+---
+
+**Beware**: This functionality is still very new and subject to change.
+
+When we say `migration` we mean _code that moves the environment from state `A` to state `B`_.
+
+You would typically add this code while implementing a new version of your system and then run it during the deployment process.
+
+We will go through the following steps:
+
+1. Create .Net Core project
+2. Add business logic
+3. Add migration for db seeding
+4. Setup `MigrationRunner` so we can run stuff
+5. Run it
+
+This guide assumes that you are on Linux.
+
+
+## 1. Create .Net Core project
+
+We will create a simple web API using standard .Net Core.
+
+```shell
+$ cd ~
+$ mkdir rapidcore-migrations
+$ cd rapidcore-migrations
+$ dotnet new webapi
+$ dotnet add package RapidCore.Mongo
+$ dotnet restore
+```
+
+You should now have a directory in your home folder containing a basic web api with a `Program.cs`, `Startup.cs` and a bunch of other files.
+
+
+## 2. Add business logic
+
+Our "business" logic will be a list of quotes. We will represent those with the `Quote` class in the root namespace.
+
+```csharp
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+
+namespace rapidcore_migrations
+{
+    public class Quote
+    {
+        [BsonIgnoreIfDefault]
```

Comment:
I would say no in this case, as this is a getting started guide that should not distract the user with other information than necessary.